AeroBody/SolarArray/SOPs.md

```diff
@@ -1,8 +1,12 @@
 SOPs
 ---
 
+[Here](Documents/Multimeter_Instructions.pdf) are the instructions for how to use the multimeter 
+
 Below are the Standard Operating Procedures (SOPs) for Solar Array manufacturing. Make sure you review them as needed before coming into the workshop! 
 
-[Encapsulation SOP](https://dochub.com/bookworm142013/orO7lgeVLJqqWW1RjMP2p5/encapsulation-sop-pdf?dt=igREcRp1CEz3hhAWkhU3)
+[Encapsulation SOP](Documents/Encapsulation_SOP.pdf)
```

Comment:
pdf link yields a 404 error. Move the pdf into the `_static` folder and update the link here.